fontforge/splineutil.c

```diff
@@ -6320,9 +6320,9 @@ EncMap *EncMapCopy(EncMap *map) {
     /* Ensure all memory available, otherwise cleanup and exit as NULL */
     if ( (new=chunkalloc(sizeof(EncMap)))!=NULL ) {
 	*new = *map;
-	if ( (new->map=malloc(map->enccount*sizeof(int32)))!=NULL ) {
+	if ( (new->map=malloc(map->encmax*sizeof(int32)))!=NULL ) {
 	    if ( (new->backmap=malloc(map->backmax*sizeof(int32)))!=NULL ) {
-		memcpy(new->map,map->map,map->enccount*sizeof(int32));
+		memcpy(new->map,map->map,map->encmax*sizeof(int32));
```

Comment:
Quibble, but this one is still better as `enccount`. `enccount` is the number of entries in the map, `encmax` is the number of entries the map has room for. There's no reason to `memcpy` the non-entries after `enccount` into the map copy.